src/set_theory/game/pgame.lean

```diff
@@ -3,9 +3,8 @@ Copyright (c) 2019 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Reid Barton, Mario Carneiro, Isabel Longbottom, Scott Morrison
 -/
-import data.fin.basic
-import data.nat.cast
-import logic.embedding
+
+import set_theory.ordinal.arithmetic
```

Comment:
Let's make a new file, `set_theory/game/birthday.lean`, rather than boost the required imports.

Reply:
Birthdays are a pretty basic notion for pre-games. In fact, they're how Conway induction is usually stated (though we're able to circumvent this by using inductive types).

Since there's a bunch of other things about combinatorial games that also build up on ordinals (particularly when we start talking about surreal numbers), I'd argue that strengthening the import is the best choice here.

Reply:
I disagree; there's almost no cost to splitting this into a separate file (the new material is right at the end of the existing file), and keeping imports minimal increases readability and compile time.

Reply:
I do think there's an organizational cost to splitting files all willy-nilly. But you do mention that there's a ton of theorems on birthdays, so the length can probably make up for this.

How do I deal with my `ordinal.to_pgame` PR, though?

Reply:
That content should go in `set_theory/game/ordinal.lean`.

Reply:
I've moved this to a new file.